integrations/larky/crypto/SHA1/test-sha1.star

```diff
@@ -0,0 +1,42 @@
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+load("@stdlib//hashlib", "hashlib")
+load("@stdlib//binascii", "binascii")
+load("@vgs//vault", "vault")
+load("@vgs//http/request", "VGSHttpRequest")
+load("@stdlib//builtins", builtins="builtins")
+load("@stdlib//base64", base64="base64")
+load("@stdlib//json", json="json")
+load("@stdlib//larky", larky="larky")
```

Comment:
Looks like hashlib, binascii, base64, larky, and operator aren't needed here. Let's remove them so we don't confuse people who are new to Larky.

Reply:
Done for both YAML and STAR